pyk/src/pyk/kore/syntax.py

```diff
@@ -255,7 +255,7 @@ def dict(self) -> Dict[str, Any]:
 
     @property
     def text(self) -> str:
-        return self.name + ' ' + _braced(sort.text for sort in self.sorts)
+        return self.name
```

Comment:
This change in how `SortApp.text` works is needed in order to get these tests passing, because the frontend does not want to parse sorts with `{}` behind them: https://github.com/runtimeverification/k/issues/2762

Reply:
Thankfully this breaks some tests in the kore unparser test-suite, so that's good :)